src/Core/Components/Nav/FluentNavItem.razor.cs

```diff
@@ -0,0 +1,364 @@
+        .AddClass("fluent-navsubitem", _isSubItem)
+        .AddClass("disabled", Disabled)
+        .Build();
+
+    /// <summary />
+    protected string? StyleValue => DefaultStyleBuilder
+        .Build();
+
+    /// <summary>
+    /// Gets or sets the icon of the nav menu item.
+    /// </summary>
+    [Parameter]
+    public Icon? Icon { get; set; }
+
+    /// <summary>
+    /// Gets or sets the href of the link.
+    /// </summary>
+    [Parameter]
+    public string? Href { get; set; }
+
+    /// <summary>
+    /// The callback to invoke when the item is clicked.
+    /// </summary>
+    [Parameter]
+    public EventCallback<MouseEventArgs> OnClick { get; set; }
+
+    /// <summary>
+    /// If true, the item will be disabled.
+    /// </summary>
+    [Parameter]
+    public bool Disabled { get; set; }
+
+    /// <summary>
+    /// Gets or sets the target attribute that specifies where to open the group, if Href is specified.
+    /// Possible values: _blank | _self | _parent | _top.
+    /// </summary>
+    [Parameter]
+    public string? Target { get; set; }
+
+    /// <summary>
+    /// Gets or sets the class names to use to indicate the item is active, separated by space.
+    /// </summary>
+    [Parameter]
+    public string ActiveClass { get; set; } = "active";
+
+    /// <summary>
+    /// Gets or sets how the link should be matched.
+    /// Defaults to <see cref="NavLinkMatch.Prefix"/>.
+    /// </summary>
+    [Parameter]
+    public NavLinkMatch Match { get; set; } = NavLinkMatch.Prefix;
+
+    /// <summary>
+    /// Gets or sets the tooltip to display when the mouse is placed over the item.
+    /// </summary>
+    [Parameter]
+    public string? Tooltip { get; set; }
+
+    /// <summary>
+    /// Gets or sets the content of the nav menu item.
+    /// </summary>
+    [Parameter]
+    public RenderFragment? ChildContent { get; set; }
+
+    /// <summary>
+    /// Gets or sets the parent <see cref="FluentNav"/> component for this instance.
+    /// </summary>
+    /// <remarks>This property is typically set automatically by the Blazor framework when the component is
+    /// used within a <see cref="FluentNav"/>. It enables the component to access shared state or functionality from
+    /// its parent navigation menu.</remarks>
+    [CascadingParameter]
+    public required FluentNav Owner { get; set; }
+
+    /// <summary>
+    /// Gets or sets the parent <see cref="FluentNavCategory"/> component for this instance.
+    /// </summary>
+    /// <remarks>This property is typically set automatically by the Blazor framework when the component is
+    /// used within a <see cref="FluentNav"/>. It enables the component to access shared state or functionality from
+    /// its parent navigation menu.</remarks>
+    [CascadingParameter(Name = "Category")]
+    public FluentNavCategory? Category { get; set; }
+
+    /// <summary />
+    [Inject]
+    public required NavigationManager NavigationManager { get; set; }
```

Comment:
Can you move all injected properties before all other properties (top of the file)?